python/remote-control-center/packaged.yaml

```diff
@@ -0,0 +1,66 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: An AWS Serverless Specification template describing your function.
+Globals:
+  Function:
+    Timeout: 30
+    Runtime: python3.7
+    Handler: lambda_function.lambda_handler
+Resources:
+  RetrieveConfigLambda:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: s3://cf-templates-shk3r44gr6qq-us-east-1/ca69335269d13c8dd4563af2b4c5b9e1
```

Comment:
This is an arbitrary s3 bucket - can we either reference from SAM or with a less arbitrary bucket?

Reply:
packaged.yaml is part of sam deployment for an account, removing from repo